src/main/java/com/example/onlinebookstore/controller/BookController.java

```diff
@@ -35,11 +36,13 @@ public BookDto getBookById(@PathVariable Long id) {
 
     @ResponseStatus(HttpStatus.CREATED)
     @PostMapping
```

Comment:
Why there is no annotation PreAuthorize on endpoints that should be accessible to user

Reply:
Added